cidc_api/models.py

```diff
@@ -69,18 +80,40 @@ class Users(CommonColumns):
     first_n = Column(String)
     last_n = Column(String)
     organization = Column(Enum(*ORGS, name="orgs"))
+    registration_approved = Column(
+        Boolean, nullable=False, default=False, server_default="false"
+    )
+    registration_approval_date = Column(DateTime)
```

Comment:
wouldn't just one column `registration_approval_date` be enough to check for approval?

Reply:
Definitely. I was replicating the user model from the old API [here](https://github.com/CIMAC-CIDC/cidc-api/blob/1877907119efd30c7698ecc7c5da84f877ba02d7/schemas/user_schema.py#L33), which has both `approved` and `registration_approval_date`. I'll make this simplification, though.